sphinx/builders/text.py

```diff
@@ -17,7 +17,7 @@
 from sphinx.builders import Builder
 from sphinx.util import logging
 from sphinx.util.osutil import ensuredir, os_path
-from sphinx.writers.text import TextWriter, TextTranslator
+from sphinx.writers.text import TextWriter, TextTranslator, MAXWIDTH
```

Comment:
I feel no reason to define `MAXWIDTH` on `sphinx.writers.text`.
How about moving to here?